code/src/hardware_usart.cpp

```diff
@@ -1,17 +0,0 @@
-#include <hardware_usart.hpp>
```

Comment:
this file was once added again because an issue #25 please specify on why you brought it back